### DISCUSSION TITLE
Please make these request in the future updates.

### DISCUSSION THREAD

**Discussion by MarkE-Official** [Zrips/CMI#6684] [Feature Requests]
CMI is almost perfect, I hope CMI could have a GUI Creator for people who wants to make a custom GUI.

**Comment by mrfloris** [comment]
Requests should be posted in the Discussion section here on github, sorry :) 
Yes, a bossShopPro like gui creator for a- the existing guis and b- our custom ones, would be rather fantastic.

https://github.com/Zrips/CMI/discussions/2944
(and a few more existing discussions) exist
